### PR TITLE
add parse-solver-output.yaml

### PR DESCRIPTION
**Pull request by bissenbay**
Combines `parse-solver-inputs` and `parse-solved-package` tasks into `parse-solver-output` task.
Depends-On: https://github.com/thoth-station/workflow-helpers/pull/107

### CODE REVIEW

**Comment by harshad16** [comment]
@pacospace  please have a look at this.
/assign @pacospace

**Comment by pacospace** [comment]
@bissenbay very well done. Perfect!! We need to have new workflow-helpers first, as these changes affect both test and stage environment.

**Comment by bissenbay** [comment]
ping @pacospace :smile:

**Comment by pacospace** [comment]
> ping @pacospace

We need to change these in test only first, as you did for package release job cronjob template, so that we can test first and then if everything work in test we will open other PR to move templates and workflow-helpers image also to stage

**Comment by bissenbay** [comment]
> > ping @pacospace
> 
> We need to change these in test only first, as you did for package release job cronjob template, so that we can test first and then if everything work in test we will open other PR to move templates and workflow-helpers image also to stage

Thanks, https://github.com/thoth-station/thoth-application/pull/482/commits/540555fe983a8358dbb702c99af1a4cab9fa5a9b

File: solver/base/kustomization.yaml [hunk 1]
```diff
@@ -6,7 +6,3 @@ commonLabels:
   app.kubernetes.io/managed-by: aicoe-thoth-devops
 resources:
   - imagestreams.yaml
```
Comment (pacospace) [inline solver/base/kustomization.yaml:8]: All changes in base needs to remain as before, otherwise we break what is in stage already

Comment (harshad16) [reply solver/base/kustomization.yaml:8]: ping @bissenbay

Comment (bissenbay) [reply solver/base/kustomization.yaml]: thanks, https://github.com/thoth-station/thoth-application/pull/482/commits/eb866721ff12944dd10db2c7b72d0cdb452116e0

File: solver/overlays/test/kustomization.yaml [hunk 2]
```diff
@@ -1,8 +1,10 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
-  - ../../base
   - thoth-notification.yaml
+  - openshift-templates/solve_and_sync-template.yaml
+  - argo-workflows/solver-template.yaml
+  - argo-workflows/parse-solver-output.yaml
```
Comment (harshad16) [inline solver/overlays/test/kustomization.yaml:7]: Please move them under: `patchesStrategicMerge:`

Comment (harshad16) [reply solver/overlays/test/kustomization.yaml:7]: Also please copy over the openshift-template to test, it has to be tested
please don't move it . https://github.com/thoth-station/thoth-application/pull/482/files#diff-6da397dde79165a7081b95c1cd493ba6

**Review by harshad16 — APPROVED** [review]
lgtm
thanks

**Pull request merged**